css/src/components/timeline.scss

```
@@ -0,0 +1,82 @@
+@use 'sass:math';
+$timeline-content-font-size: $font-size-8 !default;
+$timeline-timestamp-font-size: $font-size-9 !default;
+$timeline-timestamp-font-weight: $weight-semilight !default;
+$timeline-description-padding-top: $spacer-4 !default;
+$timeline-description-line-height: $line-height-normal;
+
+$timeline-item-border-left: $border-width solid $border !default;
+$timeline-item-min-height: $layout-6 !default;
+$timeline-item-padding-bottom: $spacer-7 !default;
+$timeline-item-padding-left: $spacer-8 !default;
+
+$timeline-item-template-gap: 8px !default;
+$timeline-item-template-width: 100% !default;
+
+$timeline-item-badge-transform-ltr: translate(
+	calc(-#{$timeline-item-padding-left} - 50%),
+	-#{math.div($timeline-timestamp-font-size, 2)}
+);
+$timeline-item-badge-transform-rtl: translate(
+	calc(#{$timeline-item-padding-left} + 50%),
+	-#{math.div($timeline-timestamp-font-size, 2)}
+);
+
+.timeline {
+	font-size: $document-font-size;
+	container-type: inline-size;
+
+	.timeline-item {
+		display: flex;
+		position: relative;
+		padding-block-end: $timeline-item-padding-bottom;
+		padding-inline-start: $timeline-item-padding-left;
+		border-inline-start: $timeline-item-border-left;
+
+		&:last-child {
+			border-inline-start: none;
+		}
+
+		.timeline-item-badge {
+			position: absolute;
+			transform: $timeline-item-badge-transform-ltr;
+
+			[dir='rtl'] & {
```

@jim-benson Based on [this comment](https://github.com/microsoft/atlas-design/pull/722#discussion_r1847019178), could you please swap this out:

```suggestion
			&:dir(rtl) {
```

---

updated